### PR TITLE
Drop no longer required `gitian-verify.py`

### PR DESCRIPTION
**Pull request by hebasto**
The last gitian built version, which is 0.21, has reached its [EOL](https://bitcoincore.org/en/lifecycle/).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -276,39 +276,37 @@ Make a new release tag, performing a few checks.
...
+pip3 install --user gpg
 ```
-(or install the distribution package, in Debian/Ubuntu this is `python3-yaml` and `python3-gpg`)
 
-The `gpg` module requires the [gpgme](https://www.gnupg.org/software/gpgme/index.html) library
-which is usually present on Linux, and can be installed with `brew install gpgme` on macOS.
+(or install the distribution package, in Debian/Ubuntu this is `python3-gpg`).
 
-Example usage: `./gitian-verify.py -r 0.21.0rc5 -d ../gitian.sigs -k ../bitcoin/contrib/builder-keys/keys.txt`
+Example usage: `./guix-verify.py -r 23.0rc2 -d ../guix.sigs -k ../bitcoin/contrib/builder-keys/keys.txt`
```
Comment (fanquake) [inline README.md:295]: Not really a blocker, but would you like to update this to use 24.0?

Comment (hebasto) [reply README.md]: Sure! Updated.

Comment (hebasto) [reply README.md]: Should be OK now. Sorry for my carelessness.

**Review by fanquake — APPROVED** [review]
ACK 0a17698563bc2367408812b130c170a7c8272bdf

**Pull request merged**